woodwork/accessor_utils.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
+
+from woodwork.logical_types import Datetime
+from woodwork.type_sys.utils import _get_ltype_class
+from woodwork.utils import _get_column_logical_type
+
+
+def _update_dtype(series, logical_type):
```

Comment:
Would it be possible to take `_update_column_dtype` from `table_accessor.py` and also use it here? I get that we're pulling out the LatLong logic to handle in #568, but I'm not sure I understand what needs to change. 

Maybe we'd remove the Ordinal data validation since that doesn't need to happen in this case (and maybe the data validation doesn't belong in the function that updates the series to match the dtype??), but it seems to me like it'd work.

Reply:
I think we can use a singular function for both. I pulled this code from the original DataColumn code, not realizing it was already pulled out into the `table_accessor.py` file. I will clean that up so they are all using the same dtype update function.

Reply:
@tamargrey I updated this to use a single `_convert_column_dtype` function for both table and column conversions now. Thanks for pointing that out!

Reply:
Also, for now I left the Ordinal validation in - it's also in the `_validate_logical_type` method in the column accessor. Need to take a closer look at that, to see if there is a better way to handle that.

Reply:
Awesome! Glad we're able to reuse so much code. I guess we can take Ordinal validation into account when we get to #515 as it's double validation between the `init_series` and Series Accessor init

Reply:
Also, I like that name `_convert_column_dtype`!

Reply:
Yeah, sounds good. I made a comment about the Ordinal stuff in Issue #515 so we don't forget about it.